lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -27,15 +27,9 @@ def initialize(build_mechanism,
       @skip_ci_status = skip_ci_status
     end
 
-    def deploy_cli_hook(parser)
-      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs', FalseClass) do |value|
-        @skip_ci_status = value
-      end
-    end
-
     def build_cli_hook(parser)
-      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs', FalseClass) do |value|
-        @skip_ci_status = value
+      parser.on('-s', '--skip-ci-status', 'Skips checks on CI jobs') do |value|
```

Comment:
I wonder if `-s` is verbose enough to be used in this case, but I'm not sure if we can get rid of that 🤔

Reply:
Fixed